pytorch_lightning/trainer/trainer.py

```diff
@@ -262,11 +262,15 @@ def __init__(
 
             max_epochs: Stop training once this number of epochs is reached. Disabled by default (None).
                 If both max_epochs and max_steps are not specified, defaults to ``max_epochs`` = 1000.
+                To disable automatic stopping, you can set ``max_epochs = -1`` and set ``max_steps`` as ``None``
+                or ``-1``. Note that if the the ``max_time`` limit is specified, it will still be observed.
```

Comment:
let's keep this short and just mention that -1 means max is disabled.

let's add the extensive user guide to the trainer.rst docs.

Reply:
Is updated change good?